src/libs/elektra/keyset.c

```diff
@@ -1955,6 +1955,10 @@ static const char * elektraOpmphmGetString (void * data)
 
 static int elektraLookupBuildOpmphm (KeySet * ks)
 {
+	if (ks->size > KDB_OPMPHM_MAX_N)
```

Comment:
Can you add some docu please, especially about preconditions (e.g. is it allowed that an Opmphm already is part of the KeySet?)

What if the KeySet already had a Opmphm and now it got larger?